metabase/metabase
# Add vertical legends with Popover

Makes LineAreaBarChart legend adaptive by switching from horizontal to vertical layout when there is not enough space. Not visible items are rendered in a Popover.

How to test:
1. Create a line chart and add it to a dashboard
2. Breakout by category - this should split the chart into multiple series
3. When there are a lot of labels, they should renderer vertically with a button to open Popover with other labels.

<img width="723" alt="Screen Shot 2021-08-12 at 5 29 59 pm" src="https://user-images.githubusercontent.com/8542534/129215146-76b01b2d-7f53-4ce5-8962-a48d954b942f.png">
<img width="761" alt="Screen Shot 2021-08-12 at 5 30 14 pm" src="https://user-images.githubusercontent.com/8542534/129215155-3636668f-ff1b-4070-a798-b4effbc85455.png">
